api-server/src/server/middleware.json

```diff
@@ -39,13 +39,16 @@
     "./middlewares/constant-headers": {},
     "./middlewares/csp": {},
     "./middlewares/flash-cheaters": {},
+    "./middlewares/version-handler": {},
+    "./middlewares/version-set-cookie": {},
     "./middlewares/passport-login": {}
   },
   "files": {},
   "final:after": {
     "./middlewares/sentry-error-handler": {},
     "./middlewares/csurf-error-handler": {},
     "./middlewares/error-handlers": {},
+    "./middlewares/version-handler": {},
```

Comment:
Do we need to hook in two times during the lifecycle of a request? I see that this handler is added to two events.